tests/test_orient_symmetric.py

```diff
@@ -119,25 +120,10 @@ def test_estimate_rotations(n_img, L, order, dtype):
     # angular distance between them (in degrees).
     Q_mat, flag = register_rotations(rots_est, rots_gt_sync)
     regrot = get_aligned_rotations(rots_est, Q_mat, flag)
-    ang_dist = np.zeros(n_img, dtype=dtype)
-    for i in range(n_img):
-        ang_dist[i] = (
-            Rotation.angle_dist(
-                regrot[i],
-                rots_gt_sync[i],
-                dtype=dtype,
-            )
-            * 180
-            / np.pi
-        )
+    mean_ang_dist = Rotation.mean_angular_distance(regrot, rots_gt_sync) * 180 / np.pi
 
     # Assert mean angular distance is reasonable.
-    if order == 2:
-        assert np.mean(ang_dist) < 4
-    elif order == 3 or order == 4:
-        assert np.mean(ang_dist) < 2
-    else:
-        assert np.mean(ang_dist) < 5
+    assert mean_ang_dist < 3
```

Comment:
That came out nice, thanks.

Reply:
I thought so too. All cases improved, but I figured 3 degrees was a reasonable tolerance and covered the weakest links.